runtime/common/src/remove_pallet.rs

```diff
@@ -77,8 +84,12 @@ use sp_std::{marker::PhantomData, vec::Vec};
 /// a multi-block scheduler currently under development which will allow for removal of storage
 /// items (and performing other heavy migrations) over multiple blocks.
 /// (https://github.com/paritytech/substrate/issues/13690)
-pub struct RemovePallet<P: Get<&'static str>>(PhantomData<P>);
-impl<P: Get<&'static str>, DbWeight: Get<DbWeight>> frame_support::traits::OnRuntimeUpgrade for RemovePallet<P> {
+pub struct RemovePallet<P: Get<&'static str>, DbWeight: Get<RuntimeDbWeight>>(
```

Comment:
We could also move it to Substrate… :man_shrugging:  
Not sure if it is worth it right now though.

Reply:
Probably `frame-support`.